src/net_processing.cpp

```diff
@@ -134,7 +134,7 @@ static constexpr double BLOCK_DOWNLOAD_TIMEOUT_PER_PEER = 0.5;
 /** Maximum number of headers to announce when relaying blocks with headers message.*/
 static const unsigned int MAX_BLOCKS_TO_ANNOUNCE = 8;
 /** Maximum number of unconnecting headers announcements before DoS score */
-static const int MAX_UNCONNECTING_HEADERS = 10;
```

Comment:
Would you explain what are unconnecting headers? My understanding is that these headers are for blocks which are received via an `inv` message but do not have any parent and belong to no chain. Is this correct?

Reply:
`nUnconnectingHeaders` tracks the number of `headers` messages (with fewer than `MAX_BLOCKS_TO_ANNOUNCE` headers) in which the first header did not connect to any known chain.